.github/workflows/review-application.yml

```
@@ -0,0 +1,29 @@
+name: Review application
+
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  review-application:
+    runs-on: ubuntu-latest
+    if: contains(github.event.issue.labels.*.name, 'application')
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Fetch processor
+        uses: dsaltares/fetch-gh-release-asset@master
```

Let's lock this GH action to a version. Using `@master` means a major version upgrade can randomly break this action and cause security issues ([good writeup here](https://github.com/actions/upload-artifact/issues/41)).
```suggestion
        uses: dsaltares/fetch-gh-release-asset@1.1.1
```